### DISCUSSION TITLE
Quick Question on the editor

### DISCUSSION THREAD

**Discussion by jryebread** [google/model-viewer#2738] [Q&A]
Hi, love model viewer! I loaded up the example following the instructions on the map model viewer page on local host, but i cannot open the model editor? is that not part of the main project, somewhere else?

Thanks!

**Comment by elalish** [comment]
Ah yes, we move some of the pages around when we deploy to github pages to shorten the URLs. If you're working locally, the links are wrong, so you have to manually navigate to `packages/space-opera/editor`.
